exercises/concept/remote-control-competition/src/test/java/RemoteControlCarTest.java

```diff
@@ -35,6 +45,8 @@ public void experimentalCarTravels20UnitsPerDrive() {
     }
 
     @Test
+    @Tag("task:3")
+    @DisplayName("The getDistanceTravelled method returns different distances for production and experimental car")
```

Comment:
This test actually verifies the implementation of the `TestTrack.race()` method, not the `RemoteControlCar.getDistanceTravelled()` method. I suggest naming it

```
The TestTrack.race method uses the drive method on the remote control car
```